Makefile

```
@@ -1,6 +1,7 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= aldaircoronel/controller:latest
+IMG ?= ghcr.io/azure/eraser-manager:latest
```

Did we want this to be something other than latest?

```suggestion
COMMIT := $(shell git rev-parse --short HEAD)
DIRTY := $(shell [ -n "$(git status --porcelain)" ] && echo "-dirty")
IMG ?= ghcr.io/azure/eraser-manager:$(COMMIT)$(DIRTY)
```

---

this will get overridden with `IMG` value (including tag) when a release is tagged

---

example: https://github.com/sozercan/eraser/runs/3435922240?check_suite_focus=true